ProjectLighthouse.Servers.Website/Pages/UserPage.cshtml.cs

```diff
@@ -18,44 +19,81 @@ public class UserPage : BaseLayout
 
     public bool CommentsEnabled;
 
-    public bool IsProfileUserHearted;
+    public List<SlotEntity>? HeartedSlots;
 
     public bool IsProfileUserBlocked;
 
+    public bool IsProfileUserHearted;
+
     public List<PhotoEntity>? Photos;
-    public List<SlotEntity>? Slots;
 
-    public List<SlotEntity>? HeartedSlots;
+    public UserEntity? ProfileUser;
     public List<SlotEntity>? QueuedSlots;
+    public List<SlotEntity>? Slots;
+
+    public bool ProfilePrivate;
+    public bool SlotsPrivate;
+    
+    public bool CommentsDisabledByModerator;
 
-    public UserEntity? ProfileUser;
     public UserPage(DatabaseContext database) : base(database)
-    {}
+    { }
 
     public async Task<IActionResult> OnGet([FromRoute] int userId)
     {
         this.ProfileUser = await this.Database.Users.FirstOrDefaultAsync(u => u.UserId == userId);
         if (this.ProfileUser == null) return this.NotFound();
 
         // Determine if user can view profile according to profileUser's privacy settings
-        if (this.User == null || !this.User.IsAdmin)
+        if (this.User == null || !this.User.IsModerator)
```

Comment:
I don't know if this was intentional or not but on the UserPage you let mods bypass privacy settings but on the SlotPage you only let admins bypass the privacy settings